engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action.controller.js

```diff
@@ -30,10 +30,10 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionContro
             working: false
         };
 
-        $scope.setState = function (working, success, errors) {
+        $scope.setState = function (working, success, error) {
             $scope.state.working = working;
             $scope.state.successMessages = success;
-            $scope.state.errorMessages = errors;
+            $scope.state.errorMessages = error;
```

Comment:
It looks like content host bulk actions notifications happen in a details page so wouldn't need to use GlobalNotification